src/status_im/contexts/wallet/home/view.cljs

```diff
@@ -94,9 +94,10 @@
                                  (rf/dispatch [:wallet/clean-send-data])
                                  (when-not multiple-accounts?
                                    (rf/dispatch [:wallet/switch-current-viewing-account
-                                                 first-account-address]))
-                                 (rf/dispatch [:wallet/start-bridge])
+                                                 first-account-address])
+                                   (rf/dispatch [:wallet/start-bridge]))
```

Comment:
Will bridge be started when there are multiple accounts? Looks like it was moved under `(when-not multiple-accounts?)` condition

Reply:
Yes @vkjr, when there are multiple accounts it will set trx type to bridge and navigate to select accounts screen, and after select account it will start the bridge.

Reply:
I see, thanks for the explanation!
Btw, what do you think about putting the info above as a comment to that code section? 
On recent sync we were discussing that a lot of information is in our heads and we can put more in the comments to make developer lives easier.
This piece of information seems to be just right for adding to the comment to help the next reader :)

Reply:
added